internal/proxy/task_query.go

```diff
@@ -240,6 +240,31 @@ func (t *queryTask) createPlan(ctx context.Context) error {
 	return nil
 }
 
+func (t *queryTask) CanSkipAllocTimestamp() bool {
+	var consistencyLevel commonpb.ConsistencyLevel
+	useDefaultConsistency := t.request.GetUseDefaultConsistency()
+	if !useDefaultConsistency {
+		consistencyLevel = t.request.GetConsistencyLevel()
+	} else {
+		collID, err := globalMetaCache.GetCollectionID(context.Background(), t.request.GetDbName(), t.request.GetCollectionName())
```

Comment:
cache schema into task object

Reply:
`collectionInfo` only contains the consistency level, not the schema.